python/flask/access-request-in-wrong-handler.yaml

```diff
@@ -0,0 +1,43 @@
+rules:
+  - id: avoid-accessing-request-in-wrong-handler
+    patterns:
+      - pattern-either:
```

Comment:
I think this would be simplified greatly by using 
```
- pattern-inside: |
   @app.route(..., method="GET")
    def $X(...):
        ...
```
followed by `pattern-either`

Reply:
Sounds good